spec/features/animals/create_spec.rb

```diff
@@ -13,7 +13,7 @@
     visit new_animal_path
 
     within('form') do
-      fill_in 'animal_pii_tag', with: "An Animal"
+      fill_in 'animal_tag', with: "An Animal"
```

Comment:
So that our tests more closely match real world usage, thoughts on this change?
```suggestion
      fill_in 'animal_tag', with: "G127"
```